index.js

```diff
@@ -647,7 +647,7 @@ function getPrograms(cb) {
 log.debug('discover interfaces');
 discover(config.ccuAddress, {
     // Todo... cuxd: {port: 8701, protocol: 'binrpc'},
-    rfd: {port: 2001, protocol: 'binrpc'},
+    rfd: {port: 2001, protocol: (config.protocolPreferXmlrpcForRfd ? 'xmlrpc' : 'binrpc') },
```

Comment:
I'm ok with that, doesn't hurt anyone. But I have to ask again :) Why would you want to use XMLRPC in favor of BINRPC? That makes every RPC call guessed 10+ times more expensive and the network traffic is guessed 100+ times bigger. BINRPC is imho the better protocol when talking to rfd/hs485d, I only see benefits, no downsides.

Reply:
You've the greater experience in rfd related knowledge, my thought was just "if they invented a new interface, they're probably not maintaining the old one as good as the new"

Reply:
I would definitely advice to use BINRPC with rfd, but I would say we include the new `protocolPreferXmlrpcForRfd` option. Or perhaps better a `protocolPreferXmlrpc`that affects hs485d also. Even if I strongly advice BINRPC usage it could be useful e.g. for wiresharking (binrpc dissector anyone?;), proxying or smth.